### PR TITLE
DOCSUP-4706 edit and translate

### PR DESCRIPTION
**Pull request by damozhaeva**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Documentation (changelog entry is not required)

Detailed description / Documentation draft:

- edit the Russian version of the text
- translate the text to English

### CODE REVIEW

File: docs/ru/operations/system-tables/errors.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# system.errors {#system_tables-errors}
+
+Содержит коды ошибок с указанием количества срабатываний.
+
+Столбцы:
+
+-   `name` ([String](../../sql-reference/data-types/string.md)) — название ошибки (`errorCodeToName`).
+-   `code` ([Int32](../../sql-reference/data-types/int-uint.md)) — номер кода ошибки.
+-   `value` ([UInt64](../../sql-reference/data-types/int-uint.md)) — число, возникающих ошибок.
```
Comment (adevyatova) [inline docs/ru/operations/system-tables/errors.md:9]:
```suggestion
-   `value` ([UInt64](../../sql-reference/data-types/int-uint.md)) — количество ошибок.
```

File: docs/ru/sql-reference/data-types/date.md [hunk 2]
```diff
@@ -9,4 +9,39 @@ toc_title: Date
 
 Дата хранится без учёта часового пояса.
 
+## Примеры {#examples}
+
+**1.** Создание таблицы со столбцом типа `DateTime` и добавление в неё данных:
```
Comment (adevyatova) [inline docs/ru/sql-reference/data-types/date.md:14]: Эта фраза избыточна. И к тому же, неправильная) Столбец в таблице имеет тип Date.

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -726,7 +726,7 @@ log_queries=1
 
 ## log_queries_min_query_duration_ms {#settings-log-queries-min-query-duration-ms}
 
-Minimal time for the query to run to get to the following tables:
+Minimum query execution time to navigate to the following tables:
```
Comment (alexey-milovidov) [inline docs/en/operations/settings/settings.md:729]: Wrong.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

File: docs/en/operations/settings/settings.md (hunk 3 above)
Comment (adevyatova) [reply docs/en/operations/settings/settings.md:729]: @azat Fix it, please!

Comment (azat) [inline docs/en/operations/settings/settings.md:729]:
```suggestion
If enabled (non-zero), queries faster then the value of this setting will not be logged (you can think about this as a `long_query_time` for [MySQL Slow Query Log](https://dev.mysql.com/doc/refman/5.7/en/slow-query-log.html)), and this basically means that you will not find them in the following tables:
```

P.S. not sure that information about mysql should be included though

File: docs/ru/sql-reference/functions/hash-functions.md [hunk 4]
```diff
@@ -153,15 +153,18 @@ SELECT groupBitXor(cityHash64(*)) FROM table
 `URLHash(s, N)` - вычислить хэш от строки до N-го уровня в иерархии URL, без одного завершающего символа `/`, `?` или `#` на конце, если там такой есть.
 Уровни аналогичные URLHierarchy. Функция специфична для Яндекс.Метрики.
 
+## farmFingerprint64 {#farmfingerprint64}
+
 ## farmHash64 {#farmhash64}
 
-Генерирует 64-х битное значение [FarmHash](https://github.com/google/farmhash).
+Создает 64-битное значение [FarmHash](https://github.com/google/farmhash) или значение Fingerprint. `farmFingerprint64` предпочтительнее для стабильных и перемещаемых значений.
```
Comment (alexey-milovidov) [inline docs/ru/sql-reference/functions/hash-functions.md:160]: The words are very wrong.

**Pull request merged**